Dockerfile

```diff
@@ -41,7 +41,13 @@ COPY ./container/root /
 # NOTE: order of operations is important, new config had to already installed from repo (above)
 RUN sed -i "s/listen [0-9]*;/listen ${CONTAINER_PORT};/" $CONF_NGINX_SITE && \
     # Make temp directory for .nginx runtime files \
-    mkdir /tmp/.nginx
+    mkdir /tmp/.nginx && \
+    # Remove older WOFF mime-type
+    sed -i "/application\/font-woff/d" /etc/nginx/mime.types && \
+    # Add again with newer mime-type
+    sed -i "s/}/\n    font\/woff                             woff;\n&/" /etc/nginx/mime.types && \
+    # Also add mime-type for WOFF2
+    sed -i "s/}/\n    font\/woff2                            woff2;\n&/g" /etc/nginx/mime.types
```

Comment:
I figured more lines that did specific things was more legible and simpler to understand, a single-line version is probably possible here.

Reply:
@eerikson could you add the link to the nginx ticket to track as a comment before your implementation?

Reply:
Ah of course, thanks for reminding me!